sites-enabled/03-all-guide.org.conf

```
@@ -0,0 +1,31 @@
+# ------------------------------------
+#  redirect guide to .org/, .org/forum
+# ------------------------------------
+
+# map for guide URLs
+map $request_uri $guide_uri {
+  include /etc/nginx/maps/guide.map;
+  default $request_uri;
+}
+
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name guide.freecodecamp.org guide.freecodecamp.dev;
```

```suggestion
  server_name guide.freecodecamp.org;
```

This is to be removed before merge